packages/apollo-datasource-rest/src/connectors/memcached.ts

```diff
@@ -47,10 +58,12 @@ export default class MemcachedKeyValueCache implements KeyValueCache {
 
     // check "timestamp" at which tags have been invalidated
     const tags = payload.t;
-    const versions = await this.client.getMulti(tags);
-    for (const tag in versions) {
-      if (versions[tag] !== undefined && versions[tag] > payload.v) {
-        return; // tag has been invalidated, therefore cache entry not valid
+    if (tags.length !== 0) {
+      const versions = await this.client.getMulti(tags);
+      for (const tag in versions) {
+        if (versions[tag] !== undefined && versions[tag] > payload.v) {
```

Comment:
We'd need a way to deal with overflow here, because currently once we wrap around entries will never be valid again.

One benefit of using per-tag versions is that we don't have to depend on ordering. All we need to check is if versions match, and if they do not the entry is invalid. So (I think?) that gets around having to deal with overflow.